server/graphql/v2/object/Order.js

```diff
@@ -112,6 +112,17 @@ export const Order = new GraphQLObjectType({
           return models.PaymentMethod.findByPk(order.PaymentMethodId);
         },
       },
+      platformFee: {
+        type: Amount,
+        description: 'If an Order has fees on top, we should return the amount',
+        resolve(order) {
+          if (order.data.isFeesOnTop) {
+            return { value: order.data.platformFee };
```

Comment:
We should add `currency`, next to `value` otherwise it will always default to USD.

Reply:
@kewitz is right, this is why I didn't include the currency

Reply:
@sbinlondon @kewitz It's not because it's the same currency as the `Order` that we should not explicitly set the currency of the `Amount`. As a general rule, when exposing an `Amount` in the GraphQL API v2, it should always have a `currency`.